CSharp/Library/Microsoft.Bot.Builder/ConnectorEx/BotData.cs

```diff
@@ -276,6 +294,11 @@ public CachingBotDataStore(IBotDataStore<BotData> inner, CachingBotDataStoreCons
 
         public long GetCount() { return cache.GetCount(); }
 
+        public async Task<BotStateDataResult> ExportStateDataAsync(IAddress key, string continuationToken)
+        {
+            throw new NotImplementedException("ExportStateDataAsync is not supported for caching bot state store");
```

Comment:
>throw new NotImplementedException [](start = 12, length = 33)

shouldn't this delegate to the inner implementation?

Reply:
Lost in a reversion.